libfwupdplugin/fu-config.c

```diff
@@ -163,22 +163,29 @@ fu_config_reload(FuConfig *self, GError **error)
 	for (guint i = 0; i < priv->items->len; i++) {
 		FuConfigItem *item = g_ptr_array_index(priv->items, i);
 		g_autofree gchar *dirname = g_path_get_dirname(item->filename);
+		g_autoptr(GError) error_load = NULL;
+		g_autoptr(GBytes) blob_item = NULL;
 		g_debug("trying to load config values from %s", item->filename);
-		if (g_file_query_exists(item->file, NULL)) {
-			g_autoptr(GBytes) blob = fu_bytes_get_contents(item->filename, error);
-			if (blob == NULL)
-				return FALSE;
-			fu_byte_array_append_bytes(buf, blob);
+		blob_item = fu_bytes_get_contents(item->filename, &error_load);
+		if (blob_item == NULL) {
+			if (g_error_matches(error_load, G_FILE_ERROR, G_FILE_ERROR_ACCES)) {
```

Comment:
is it EACCESS for not-found as well? my UNIX-foo is poor.

Reply:
no it should be `G_FILE_ERROR_EXIST``, added another case for it.